### PR TITLE
 pythonPackages.fuzzyfinder: init at 2.1.0, caerbannog: init at 0.3 

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
Heard about it in https://cast.postmarketos.org/episode/03-Librem5-Mainlining-Feedbackd-Plasma521/.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @craftyguy

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114039` at b80207d6 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/99f97ede0664a205c47b89ead7f3e8d4">caerbannog</a></li></li>
    <li><a href="https://gist.github.com/312b2f9ba6c08e7607e0aefe37c93e4c">python37Packages.fuzzyfinder</a></li></li>
    <li><a href="https://gist.github.com/a02e9f836c956024e23f7c7eb2feaee0">python38Packages.fuzzyfinder</a></li></li>
    <li><a href="https://gist.github.com/470319f795ec427902f904d1de83bdd4">python39Packages.fuzzyfinder</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/applications/misc/caerbannog/default.nix:2:3:
```
  |
2 | , stdenv
  |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/misc/caerbannog/default.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+, libhandy
+, libnotify
+, pango
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "caerbannog";
+  version = "0.3";
+
+  format = "other";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/caerbannog/default.nix:21]:
```suggestion
  version = "0.3";
  format = "other";
```

File: pkgs/applications/misc/caerbannog/default.nix [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~craftyguy/caerbannog";
+    rev = "0.3";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/caerbannog/default.nix:25]:
```suggestion
    rev = version;
```

File: pkgs/development/python-modules/fuzzyfinder/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  version = "2.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c56d86f110866becad6690c7518f7036c20831c0f82fc87eba8fdb943132f04b";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/fuzzyfinder/default.nix:18]: Please add a pythonImportsCheck.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/fuzzyfinder/default.nix (hunk 3 above)
Comment (dotlambda) [reply pkgs/development/python-modules/fuzzyfinder/default.nix:18]: It's an application, not a module.

**Pull request merged**